testing/integration/ess/upgrade_fleet_test.go

```diff
@@ -52,6 +52,7 @@ func TestFleetManagedUpgradeUnprivileged(t *testing.T) {
 		Local: false, // requires Agent installation
 		Sudo:  true,  // requires Agent installation
 	})
+	t.Skip("Skipping test temporarily until the DRA with the code that supports windows/arm64 is produced")
```

Comment:
Let's not create more work for ourselves, let these run and we'll force merge through the failures and they'll auto correct as soon as these changes hit the build.

Reply:
@cmacknz the CI with the skip is almost done, but I have no issue reverting just the last commit and then you can power merge, your call